src/ContentScript/index.js

```diff
@@ -57,11 +59,12 @@ const ToggleReading = (enableReading) => {
 
   if (boldedElements.length < 1) {
     addStyles();
-    parseDocument();
+    [...document.body.children].flatMap((child) => [...child.childNodes]).forEach(parseNode);
```

Comment:
maybe this is good enough?

```suggestion
    [...document.body.children].forEach(parseNode);
```